dmoj/utils/ansi.py

```diff
@@ -29,3 +26,7 @@ def format_inline(text, attrs):
 
     return re.sub(r'#ansi\[(.*?)\]\((.*?)\)',
                   lambda x: format_inline(x.group(1), x.group(2)) if not no_ansi else x.group(1), text)
+
+
+def print_ansi(*args, **kwargs):
+    print(*map(ansi_style, args), **kwargs)
```

Comment:
Wait, what happens if there are non-string arguments?

Reply:
There are currently none, and I don't want to introduce more unused code. It's trivially fixable if we ever need to print non-string things.